src/bitwardenCrdOperator.py

```diff
@@ -19,6 +19,7 @@ def bitwarden_signin(logger, **kwargs):
         logger.info("BW_HOST not set. Assuming SaaS installation")
     command_wrapper(logger, "login --apikey")
     unlock_bw(logger)
+    sync_bw(logger)
```

Comment:
For some reason, I got some race condition situation where the vault was not synced. I guess it's important to do this here.